src/test/java/org/superbiz/ColorResourceTest.java

```diff
@@ -42,10 +47,11 @@
  * A second value-add is it is possible to build WebArchives that are slim and trim and therefore
  * isolate the functionality being tested.  This also makes it easier to swap out one implementation
  * of a class for another allowing for easy mocking.
- *
  */
 @RunWith(Arquillian.class)
-public class ColorServiceTest extends Assert {
+@RunAsClient
```

Comment:
Hmm. It turns out that despite the fact that Aslak and I discussed this, it doesn't seem to have ever been implemented in core. So the testable=false is still required to keep the test class from being deployed. But we will want @RunAsClient so that the tests are executed from outside the server.